.github/workflows/beta_build_and_deploy.yml

```diff
@@ -0,0 +1,26 @@
+name: Beta Build & Deploy Version
+
+on:
+  push:
+    tags:
+      - 'v*.*.*-beta'
```

Comment:
This will run the build and release steps for any tags ending in `-beta` after the semantic version.